macintacos/vspacecode-config.json

```diff
@@ -552,128 +572,449 @@
+              "command": "editor.action.formatDocument",
+              "key": "=",
+              "name": "Format document",
               "type": "command"
             },
             {
-              "command": "csv.preview",
-              "key": "v",
-              "name": "CSV Preview",
+              "command": "editor.action.formatDocument",
+              "key": "d",
+              "name": "Format document with...",
               "type": "command"
             },
             {
-              "command": "excel.preview",
-              "key": "V",
-              "name": "Excel preview",
+              "command": "editor.action.formatChanges",
+              "key": "m",
+              "name": "Format modified lines only",
+              "type": "command"
+            },
+            {
+              "command": "editor.action.formatSelection",
+              "key": "s",
+              "name": "Format selection",
+              "type": "command"
+            },
+            {
+              "command": "editor.action.formatSelection.multiple",
+              "key": "S",
+              "name": "Format selection with...",
               "type": "command"
             }
           ],
-          "key": "c",
-          "name": "CSV/Excel...",
+          "key": "=",
+          "name": "Format...",
           "type": "bindings"
         },
         {
           "bindings": [
             {
-              "command": "markdown.extension.editing.toggleMath",
-              "key": "m",
-              "name": "Toggle math",
+              "command": "go.fill.struct",
+              "key": "f",
+              "name": "Fill struct",
               "type": "command"
             },
             {
-              "command": "markdown.extension.checkTaskList",
-              "key": "c",
-              "name": "Check Task List",
+              "command": "go.add.tags",
+              "key": "t",
+              "name": "Add tags to struct fields",
               "type": "command"
             },
             {
-              "command": "markdown.extension.editing.toggleBold",
-              "key": "b",
-              "name": "Toggle bold",
+              "command": "go.remove.tags",
+              "key": "T",
+              "name": "Remove tags from struct fields",
               "type": "command"
             },
             {
-              "command": "markdown.extension.editing.toggleItalic",
+              "command": "go.import.add",
               "key": "i",
-              "name": "Toggle italic",
+              "name": "Add import",
               "type": "command"
             },
             {
-              "command": "markdown.extension.editing.toggleStrikethrough",
-              "key": "s",
-              "name": "Toggle strikethrough",
+              "command": "go.impl.cursor",
+              "key": "I",
+              "name": "Generate interface stubs",
+              "type": "command"
+            }
+          ],
+          "key": "A",
```

Comment:
It seems like `i` is often used in other layers from import, and as well for inserting, although, i understand it is a bit weird that it has a remove command

Reply:
Ah, I'm cool with changing this to "i" - I just kept seeing the word "add" and just felt like it needed to be an "a", but I can switch this over to "i" if it's a convention most folks are familiar with